storage/container_storage_interface/persistent-storage-csi-alicloud-disk.adoc

```diff
@@ -0,0 +1,23 @@
+[id="persistent-storage-csi-alicloud-disk"]
+= AliCloud Disk CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-alicloud-disk
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Alibaba AliCloud Disk Storage.
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
+
+To create CSI-provisioned PVs that mount to AliCloud Disk storage assets, {product-title} installs the AliCloud Disk CSI Driver Operator and the AliCloud Disk CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
+
+* The _AliCloud Disk CSI Driver Operator_ provides a storage class (`alicloud-disk`) that you can use to create persistent volume claims (PVCs). The AliCloud Disk CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on-demand, eliminating the need for cluster administrators to pre-provision storage.
```

Comment:
I noticed that your removed the initial capitalization from Driver Operator in modules/persistent-storage-csi-enable. Should this and any other instances also be fixed?